mlrun/datastore/targets.py

```diff
@@ -1594,10 +1600,7 @@ def as_df(
         time_column=None,
         **kwargs,
     ):
-        try:
-            query = kwargs["query"]
```

Comment:
you don't need the query in `kwargs` anymore? You could have done `query = kwargs.get("query", {})`